cleo/opto/opsin_library.py

```diff
@@ -173,6 +434,150 @@ def chr2_h134r_4s() -> BansalFourStateOpsin:
     )
 
 
+# ibroneuroscience
+
+
+def ChETA_4s() -> BansalFourStateOpsin:
+    """Returns a 4-state ChETA model.
+
+    Params given in Bansal et al., 2020.
+    Action spectrum from `Gunaydin, L., Yizhar, O., Berndt, A. et al., 2010, Fig. 2b
+    <https://doi.org/10.1038/nn.2495>`_,
+    extracted using `Plot Digitizer <https://plotdigitizer.com/>`_.
+    """
+
+    return BansalFourStateOpsin(
+        Gd1=0.21 / ms,
+        Gd2=0.02 / ms,
+        Gr0=2e-4 / ms,
+        g0=21.5 * nsiemens,
+        phim=1e16 / mm2 / second,
+        k1=3 / ms,
+        k2=1 / ms,
+        Gf0=0.02 / ms,
+        Gb0=0.015 / ms,
+        kf=0.01 / ms,
+        kb=0.01 / ms,
+        gamma=0.05,
+        p=1,
+        q=1,
+        E=0 * mV,
+        name="ChETA",
+        spectrum=[
+            (445, 0.36),
+            (452, 0.47),
+            (460, 0.61),
+            (468, 0.77),
+            (478, 0.89),
+            (490, 1),  # <https://www.addgene.org/guides/optogenetics/>
+            (500, 1),
+            (514, 0.89),
+            (530, 0.70),
+            (550, 0.42),
+            (576, 0.12),
+        ],
+    )
+
+
+def Chronos_4s() -> BansalFourStateOpsin:
+    """Returns a 4-state Chronos model.
+
+    Params given in Bansal et al., 2020.
+    Action spectrum from `Soor, Navjeevan S., et al., 2019, Fig. 1b
+    <https://doi.org/10.1088/1361-6463/aaf944>`_,
+    extracted using `Plot Digitizer <https://plotdigitizer.com/>`_.
+    """
+
+    return BansalFourStateOpsin(
+        Gd1=0.278 / ms,
+        Gd2=0.01 / ms,
+        Gr0=1.2e-3 / ms,
+        g0=39 * nsiemens,
+        phim=0.8e16 / mm2 / second,
+        k1=1.8 / ms,
+        k2=0.01 / ms,
+        Gf0=0.05 / ms,
+        Gb0=0.08 / ms,
+        kf=0.1 / ms,
+        kb=0.01 / ms,
+        gamma=0.05,
+        p=0.8,
+        q=0.9,
+        E=0 * mV,
+        name="Chronos",
+        spectrum=[
+            (388, 0.18),
+            (404, 0.21),
+            (419, 0.30),
+            (431, 0.41),
+            (439, 0.50),
+            (447, 0.57),
+            (457, 0.67),
+            (465, 0.74),
+            (472, 0.80),
+            (480, 0.86),
+            (490, 0.92),
+            (498, 1),
+            (516, 0.84),
+            (507, 0.90),
```

Comment:
here's another place the spectrum is out of order. For legibility and reproducibility, it should be in order, but I remembered I actually should have already accounted for this in `light_dependence.py`, line 125...don't know why I'm getting an error. Could you add a unit test for this? I guess it would be a new file `tests/light/test_light_dependence.py`